content/tutorials/esc-version-tagging/index.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Configure applications dynamically with ESC version tagging
+title_tag: Configure applications dynamically with ESC version tagging
+layout: single
+description: Use immutable versioning and dynamic tagging to configure multiple applications safely and easily with Pulumi ESC.
+meta_desc: Use immutable versioning and dynamic tagging to configure multiple applications safely and easily with Pulumi ESC.
+meta_image: meta.png
+weight: 999
+summary: |
+    As a hosted service of Pulumi Cloud, [Pulumi ESC](https://pulumi.com/esc) makes it easy to store and retrieve static and dynamic configuration settings --- including encrypted secrets --- manage them securely and flexibly, and use them in your applications in various ways. In this tutorial, you'll use ESC's [version tagging](/docs/esc/environments/versioning/) features to configure multiple running Node.js applications with a single shared configuration value referenced by tag name.
```

Comment:
I would remove the lead-in "As a hosted service of Pulumi Cloud," to stay consistent with our multi-product messaging. Also, remove the em dash clause and tighten up the ending by removing "in various ways".

```markdown
[Pulumi ESC](https://pulumi.com/esc) makes it easy to store and retrieve static and dynamic configuration settings and encrypted secrets, manage them securely and flexibly, and use them in your applications. In this tutorial, ...
```